framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1172,10 +1172,17 @@ protected void performFlowAnalysis(ClassTree classTree) {
 
                 while (!lambdaQueue.isEmpty()) {
                     Pair<LambdaExpressionTree, Store> lambdaPair = lambdaQueue.poll();
+                    MethodTree enclosingMethod =
+                            (MethodTree)
+                                    TreeUtils.enclosingOfKind(
+                                            getPath(lambdaPair.first), Kind.METHOD);
+                    ClassTree enclosingClass =
```

Comment:
I'm wondering whether this could just be `classTree`.
(I'm also wondering what the difference between `ct` and `classTree` is and whether this code should use `ct` instead...)